client/dashboard/sites/backups/backups-list.tsx

```diff
@@ -125,7 +130,7 @@ export function BackupsList( {
 				view={ view }
 				onChangeView={ onChangeView }
 				isLoading={ isLoadingActivityLog }
-				defaultLayouts={ { table: {} } }
+				defaultLayouts={ DEFAULT_LAYOUTS }
```

Comment:
We are not supporting the table view on the Backup page, only `list`.

```suggestion
				defaultLayouts={ { table: {} } }
```

Reply:
Updated in [51f3e28](https://github.com/Automattic/wp-calypso/pull/105961/commits/51f3e282f17f47370c0fc7c7a30da566cbe42a34)!